configure.ac

```diff
@@ -6,6 +6,7 @@ AC_INIT([Heimdal],[1.6.99],[heimdal-bugs@h5l.org])
 AC_CONFIG_SRCDIR([kuser/kinit.c])
 AC_CONFIG_HEADERS(include/config.h)
 AC_CONFIG_MACRO_DIR([cf])
+AC_USE_SYSTEM_EXTENSIONS
```

Comment:
Thanks for noting that the S11 build requires `AC_USE_SYSTEM_EXTENSIONS`.  I think what I'd like to do is build it myself (it's been a while) and see which extensions, where, and do the right thing on a case-by-case basis.  Or alternatively use `AC_USE_SYSTEM_EXTENSIONS` only on Solaris hosts.